Problems with `cryptsetup` and `libseccomp` when installing Apptainer from source

### Version of Apptainer

What version of Apptainer (or Singularity) are you using? Run

`apptainer --version` (or `singularity --version`).

`apptainer version 1.1.3`

### Expected behavior

I was installing `apptainer` (v1.1.3) from source on a `rhel8` node. I have `cryptsetup` installed (see below)
```sh
$ yum list installed cryptsetup
Not root, Subscription Management repositories not updated
Installed Packages
cryptsetup.x86_64
```

Also, `libseccomp` is installed (this one as a module) as well:
```sh
$ pkg-config --modversion  -- libseccomp
2.5.4
```

### Actual behavior

The output of `./mconfig` was the following:
```
=> project apptainer setup with :
    - host arch: x86_64
    - host wordsize: 64-bit
    - host C compiler: cc
    - host Go compiler: /apps/golang/1.19.1/go/bin/go
    - host system: unix
      ---
    - target arch: x86_64
    - target wordsize: 64-bit
    - target C compiler: cc
      ---
    - config profile: release
      ---
    - SUID install: no
    - Network plugins: yes
      ---
    - verbose: no
      ---
    - cryptsetup: 
      ---
    - version: 1.1.3
```

I proceeded with the installation, and when I ran `sudo make install` I got the following error message:
```
 GO apptainer
    [+] GO_TAGS "containers_image_openpgp sylog oci_engine apptainer_engine fakeroot_engine apparmor selinux seccomp"
# pkg-config --cflags  -- libseccomp
Package libseccomp was not found in the pkg-config search path.
Perhaps you should add the directory containing `libseccomp.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libseccomp', required by 'virtual:world', not found
pkg-config: exit status 1
make: *** [Makefile:181: apptainer] Error 2
```

Also, take a look at the `PKG_CONFIG_PATH`
```sh
$ echo $PKG_CONFIG_PATH
/apps/libseccomp/2.5.4/lib/pkgconfig:apps/gcc/11.3.0/lib/pkgconfig
```

I am unsure whether this is a bug or if having `libseccomp` as a module would be a problem here.
When we installed Singularity in the past, `libseccomp` was a module as well and everything worked smoothly.

### What OS/distro are you running

```sh
$ cat /etc/os-release
$ cat /etc/os-release 
NAME="Red Hat Enterprise Linux"
VERSION="8.7 (Ootpa)"
ID="rhel"
ID_LIKE="fedora"
VERSION_ID="8.7"
PLATFORM_ID="platform:el8"
PRETTY_NAME="Red Hat Enterprise Linux 8.7 (Ootpa)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:8::baseos"
HOME_URL="https://www.redhat.com/"
DOCUMENTATION_URL="https://access.redhat.com/documentation/red_hat_enterprise_linux/8/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

REDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 8"
REDHAT_BUGZILLA_PRODUCT_VERSION=8.7
REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="8.7"
```

### How did you install Apptainer

I'm installing from source.

--- a/mconfig
+++ b/mconfig
@@ -922,10 +922,6 @@ if [ "$verbose" = 1 ]; then
 else
 	echo "    - verbose: no"
 fi
-if test "${host}" = "unix" ; then
-	echo "      ---"
-	echo "    - cryptsetup: ${cryptsetup_path}"
-fi
 echo "      ---"
 echo "    - version: $package_version"
 

